src/test/scala/tech/cryptonomic/conseil/tezos/DatabaseConversionsTest.scala

```diff
@@ -819,14 +819,16 @@ class DatabaseConversionsTest
 
       "convert a Ballot operation to a database row" in {
 
-        val converted = (block, groupHash, Ballot: Operation).convertTo[Tables.OperationsRow]
+        val converted = (block, groupHash, sampleBallot: Operation).convertTo[Tables.OperationsRow]
 
         converted.operationId shouldBe 0
         converted.operationGroupHash shouldBe groupHash.value
         converted.blockHash shouldBe block.data.hash.value
         converted.blockLevel shouldBe block.data.header.level
         converted.timestamp shouldBe Timestamp.from(block.data.header.timestamp.toInstant)
-        converted.kind shouldBe "ballot"
+        converted.ballot shouldBe Some("yay")
```

Comment:
The newly-added fields in the operations (i.e. ballot, source), should now be added to the list of empty fields checked in all the other operations' cases.

I'm referring to the checks that go
```
forAll(
  ... a long chained list of converted.fieldname
) {
  _ shouldBe 'empty
}
```